src/main/java/org/broadinstitute/consent/http/db/ConsentMapper.java

```diff
@@ -27,10 +27,13 @@ public Consent map(int index, ResultSet r, StatementContext ctx) throws SQLExcep
         consent.setName(r.getString("name"));
         consent.setCreateDate(r.getTimestamp("createDate"));
         consent.setSortDate(r.getTimestamp("sortDate"));
-        consent.setLastUpdate((r.getTimestamp("lastUpdate") == null) ? null : r.getTimestamp("lastUpdate"));
-        consent.setTranslatedUseRestriction((r.getString("translatedUseRestriction") == null) ? null : r.getString("translatedUseRestriction"));
+        consent.setLastUpdate(r.getTimestamp("lastUpdate"));
+        consent.setTranslatedUseRestriction(r.getString("translatedUseRestriction"));
```

Comment:
minor optimizations. `getTimestamp` and `getString` return null if the sql is null.